### DISCUSSION TITLE
How to view contract's source code in a web editor directly on Etherscan

### DISCUSSION THREAD

**Discussion by nathaniel1616** [Cyfrin/foundry-full-course-cu#3119] [Q&A] [answered]
In the Account Abstraction content, Patrick was able to change the URL of the contract on Etherscan to make contract source files be in  a web editor (VSCode like in the browser) . How is that done ?

eg.
This is for Github.
When you have a repo such as ```https://github.com/Cyfrin/foundry-full-course-cu ```

we can replace  the ```.com``` to  ```.dev```  to set up a web editor.

```https://github.dev/Cyfrin/foundry-full-course-cu```

What is the similar for Etherscan . Thanks

**Comment by EngrPips** [comment, marked as answer]
Hello @nathaniel1616, While browsing smart contract code on Etherscan just change URL from . io to . deth.net . This will open Visual Studio Code instance and fetch the verified code using Etherscan API.

**Reply by nathaniel1616** [reply]
yeah that is right . Thanks so much @EngrPips 🤩🤩

**Reply by EngrPips** [reply]
Anytime, Friend. Keep crushing it.
